include/sersicmixcomponent.h

```diff
@@ -81,7 +81,8 @@ class SersicParametricEllipseHolder {
  * but deliberately excludes the very inner and outermost regions.
  *
  * @note See https://ned.ipac.caltech.edu/level5/March05/Graham/Graham2.html
- *       for a useful summary of various properties of the Sersic profile.
+ *       (Graham & Driver 2005) for a useful summary of various properties
+ *       of the Sersic profile.
```

Comment:
Prefer [inline links](https://developer.lsst.io/cpp/api-docs.html#references):
```suggestion
 * @note See [Graham & Driver 2005](https://ned.ipac.caltech.edu/level5/March05/Graham/Graham2.html)
 *       for a useful summary of various properties
 *       of the Sersic profile.
```